railties/lib/rails/application/configuration.rb

```diff
@@ -199,6 +199,7 @@ def load_defaults(target_version)
 
           if respond_to?(:action_dispatch)
             action_dispatch.return_only_request_media_type_on_content_type = false
+            action_dispatch.cookies_serializer = :json
```

Comment:
I was just working on upgrading an app to Rails 7 and hit this case. Cookie serialization changed when changing to Rails 7 defaults, but this change isn't called out on https://guides.rubyonrails.org/configuring.html#results-of-config-load-defaults so it took me quite some time to find that this is where it changed. Had to review the code itself.

Is this something that should be documented at https://guides.rubyonrails.org/configuring.html#results-of-config-load-defaults? So that it's more clear that this default changed from using `marshal` to `json` for the Rails 7 defaults?

Reply:
@dbussink there was an entry in `new_framework_defaults_7_0.rb` for this, see [here](https://github.com/rails/rails/blob/7-0-stable/railties/lib/rails/generators/rails/app/templates/config/initializers/new_framework_defaults_7_0.rb.tt#L85).

Did you keep that line commented or uncommented?

Reply:
@ghiculescu I was reading the docs at https://guides.rubyonrails.org/configuring.html#results-of-config-load-defaults to try to find out which setting broke tests for me and I couldn't find any one that applied. I didn't use that process to be honest but only looked at the documentation where it was missing.

Reply:
You're right, it should be in that guide too. I will make a PR.

Could you please see if using new framework defaults fixes the issue for you?

Reply:
@ghiculescu Once I knew what the issue was it was really simple to fix, so I think it's more a documentation issue than anything else.

Reply:
Cool, I added to docs in https://github.com/rails/rails/pull/44067

Thank you for raising this.

Reply:
@ghiculescu Thanks for the quick fix! :heart: